composer.json

```diff
@@ -21,7 +21,8 @@
         "symfony/framework-bundle": "~2.3",
         "symfony/doctrine-bridge": "~2.3",
         "phpcr/phpcr-implementation": "2.1.*",
-        "phpcr/phpcr-utils": "~1.2.0"
+        "phpcr/phpcr-utils": "~1.2.0",
+        "symfony/symfony": "2.7.*@dev"
```

Comment:
uups. i guess that happened while testing locally. will fix it in master, thanks for noticing, stof!